src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -1079,6 +1079,8 @@ public static function provideSnake()
             ['symfony', 'SYMFONY'],
             ['symfony_is_great', 'SYMFONY IS GREAT'],
             ['symfony_is_great', 'SYMFONY_IS_GREAT'],
+            ['symfony_is_great', 'symfony    is     great'],
+            ['symfony_is_great', 'SYMFONY    IS     GREAT'],
```

Comment:
can you please add a test case like 
`['symfony_is_great', 'SYMFONY _ IS _ GREAT'],`

Reply:
done, and this also revealed a bug in my patch